src/components/IndividualCategory.jsx

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getCategories, getReviews } from "../utils/API";
+import ThumbsCategory from "./ThumbsCategory";
+
+const IndividualCategory = () => {
+  const { slug } = useParams();
+  const [singleCategory, setSingleCategory] = useState({});
+  const [categoryReviews, SetCategoryReviews] = useState([]);
+  console.log(categoryReviews);
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    getCategories().then((response) => {
+      const filteredCategory = response.filter(
+        (category) => category.slug === slug
+      );
+      setSingleCategory(filteredCategory[0]);
+      setLoading(false);
+    });
+  }, [slug]);
```

Comment:
Here you're fetching all the categories to set a single one in state. As far as I can see you only use the `slug` property on this object once you have it? Unless you need it for something else that I've missed you already have the slug from the `useParams()` and can use that directly without this useEffect.

If this is the case you can take this whole thing out.